src/site/_release-instructions-xml-schema.adoc

```diff
@@ -0,0 +1,67 @@
+////
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements. See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+////
+
+// ██     ██  █████  ██████  ███    ██ ██ ███    ██  ██████  ██
+// ██     ██ ██   ██ ██   ██ ████   ██ ██ ████   ██ ██       ██
+// ██  █  ██ ███████ ██████  ██ ██  ██ ██ ██ ██  ██ ██   ███ ██
+// ██ ███ ██ ██   ██ ██   ██ ██  ██ ██ ██ ██  ██ ██ ██    ██
+//  ███ ███  ██   ██ ██   ██ ██   ████ ██ ██   ████  ██████  ██
+//
+// Below instructions are shared by all Maven-based Apache Logging Services projects.
+// Be extremely cautious while making changes!
+
+include::_constants.adoc[]
+
+[#release-instructions-xml-schema]
+== XML schema release instructions
+
+This section explains how to publish XML schemas (i.e., XSD files) of projects.
+It is crucial to understand certain things while doing an XML schema release:
+
+Projects and XML schemas have different lifecycles::
+A new release of a project does not necessarily mean a new release of its XML schemas.
+XML schemas might have been untouched, or they might contain minor changes while the project itself contains breaking changes, etc.
+Hence, the project version and the XML schema version most of the time differ.
+Check if there are indeed changes to the XML schema.
+If so, set its version accordingly.
```

Comment:
Maybe some examples are in order. In the case of Log4j:

- the configuration schema follows only the changes to `@Plugin` annotated classes. If we didn't introduce a new configuration attribute, the version should be a micro version bump.
- if a minor version bump is required (new configuration attribute), let us follow the same convention as for package versioning: we bump the version of the schema to the current Log4j version.

Reply:
In 034dd7bddbc7d64d8a8ba233dace280af497e0bf, added examples citing from Log4j Changelog XSDs and warned about the auto-generated Log4j XSD.

> if a minor version bump is required (new configuration attribute), let us follow the same convention as for package versioning: we bump the version of the schema to the current Log4j version.

I don't agree with this due to following reasons:

* Package versions are internal and help us to track breaking changes, etc. They are not exposed to users. OTOH, XSD versions are.
* _"we bump the version of the ~schema~ package to the current Log4j version"_, not because of our preference, but due to `bnd-maven-plugin` version assistance working like so.